dags/job_label.py

```diff
@@ -0,0 +1,45 @@
+import csv
+import logging
+import os
+from datetime import datetime
+import json
+import pandas as pd
+
+from airflow import DAG
+from airflow.hooks import S3Hook
+from airflow.operators import BaseOperator
+
+from utils.airflow import datetime_to_quarter
+from utils.s3 import upload
+from utils.hash import md5
+from config import config
+from datasets import job_postings
+
+from algorithms.corpus_creators.basic import JobCategoryCorpusCreator
+
+# some DAG args, please tweak for sanity
+default_args = {
+    'depends_on_past': False,
+    'start_date': datetime(2011, 1, 1),
+}
+
+dag = DAG(
+    'job_labeler',
+    schedule_interval=None,
+    default_args=default_args
+)
+
+class JobLabelOperator(BaseOperator):
+    def execute(self, context):
+        s3_conn = S3Hook().get_conn()
+        quarter = datetime_to_quarter(context['execution_date'])
+        job_label_filename = 'tmp/job_label_train_'+quarter+'.csv'
```

Comment:
It might make sense to put the tmp/ directory itself into git, with a dummy file. And then keep all other files in the directory ignored. Because we use the tmp/ directory in a bunch of places, and we want to always be sure that it's been created, where it be a unit test, somebody's laptop, or a new server installation.

In one of the other places I put the creation check in code, but there's no guarantee that particular check will be run first. Since we have multiple entry points that rely on tmp/, I prefer to make sure it gets created when the repo is cloned.